os_faults/drivers/tcpcloud.py

```diff
@@ -203,13 +216,13 @@ def verify(self):
 
     def _get_cloud_hosts(self):
         if not self.cached_cloud_hosts:
-            cmd = "salt -E '(ctl*|cmp*)' network.interfaces --out=yaml"
+            cmd = "salt -E '(infra*)' network.interfaces --out=yaml"
             result = self.execute_on_master_node({'command': cmd})
             stdout = result[0].payload['stdout']
             for fqdn, net_data in yaml.load(stdout).items():
                 host = node_collection.Host(
-                    ip=net_data['eth0']['inet'][0]['address'],
-                    mac=net_data['eth0']['hwaddr'],
+                    ip=net_data['eth1']['inet'][0]['address'],
```

Comment:
should this part of the coed be enclosed in a try block? what if the 'eth1' key does not exists? this is something carried from the code base but just wondering :)